### ISSUE TITLE
Default value for `nb_np_list` parameter.

### ISSUE THREAD

**Issue by thilinarmtb** [thilinarmtb/NekBench#18]
PR #15 set the default value of `nb_np_list` to 1 if the user did not provide any.
One problem I see with that is the given case may not run with a single MPI rank.
May be a better way to do this may be to set a `nb_np_list` value depending on
the machine. We can move the definition of the variable to appropriate machine
file and set it accordingly. For example, for `linux` machine type this can be 4,
for `cetus`, this can be 64.

@kent0 : What do you think?

**Comment by kent0** [comment]
It's true that the default value of 1 won't work if a specific machine has more than 1 core. In that case the user will have to specify the value/s anyway. It may be a good idea for it to be machine specific, but  I think it's ok for the 'default' to be 1. We may want to use a model where default values get set in go.sh, but gets overridden by sourcing a machine specific variables file. We can also follow this model and add a user configuration file that will get sourced before the tests start.

**Comment by thilinarmtb** [comment]
Yes, this is exactly what I am talking about. Let's keep this issue till we add machine
specific defaults. Currently, the defaults are for a `linux` machine type.
